### PR TITLE
Switch to GitHub Actions

### PR DESCRIPTION
**Pull request by cam-stitt**

### CODE REVIEW

File: .github/workflows/get-go-version.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Reusable Go Version Workflow
```
Comment (cam-stitt) [inline .github/workflows/get-go-version.yml:1]: Had to add this in here as it's a public repo so does not have access to the shared actions repository.

Comment (glennsarti) [reply .github/workflows/get-go-version.yml:1]: Perhaps that actions repo should be public then?

**Review by glennsarti — APPROVED** [review]
Looks green!

File: .github/workflows/get-go-version.yml (hunk 1 above)
Comment (cam-stitt) [reply .github/workflows/get-go-version.yml:1]: I'd prefer not mostly because of the *other* workflows that exist.

**Pull request merged**